4-generalization/labs/05-peoples-park/lab.qmd

```diff
@@ -35,57 +35,76 @@ In answering the following questions, it will be helpful to consult
 
-<!-- #### Question 7 -->
+> For the remaining questions in this lab, drop rows with `NA`s in the columns that you're using for that question.
 
-<!-- Create two 95% confidence intervals for the mean rating of the condition of People's Park using both the bootstrap and the normal curve. Interpret the interval in the context of the problem in a clear sentence. -->
+#### Question 5
 
-<!-- Which method for making the CI, if any, would you expect to be more accurate? -->
+What is the mean and median rating of the condition of People's Park (question 15 on the survey)?
 
-<!-- #### Question 8 -->
+#### Question 6
 
-<!-- Create two 95% confidence intervals for the overall proportion of students that support the People's Park Project without having been exposed to the information on page 14 using both the bootstrap and the normal curve. Interpret the interval in the context of the problem in a clear sentence. Do your point estimates approximately match those reported in the Chancellor's email? -->
+Create a new column called `change_in_support` that measures the change in support from question 18 to 21 of the survey. What is the mean change in support of the survey participants in each class (freshman, sophomore, etc) for the People's Park Project after reading the information? What assumption must you make about the values of the Likert scale in order for these statistics to be informative?
 
-<!-- Which method for making the CI, if any, would you expect to be more accurate? -->
+#### Question 7
 
-<!-- #### Question 9 -->
+Construct one addition visualization that captures a variable or relationship between two variables that you are interested in. Describe the structure that you see in the plot.
 
-<!-- Using just the bootstrap, create a 95% confidence intervals for the mean change in support for the Project across the entire population after being exposed to the information on page 14. -->
+#### Question 8
 
-<!-- #### Question 10 -->
+Create a 95% confidence intervals for the mean rating of the condition of People's Park using the normal curve. Interpret the interval in the context of the problem in a clear sentence.
 
-<!-- Does your interval from the previous question contain 0? What are the implications of that for those working in the Chancellor's Office on the People's Park Project? -->
+#### Question 9
+
+Create a 95% confidence interval using the normal curve for the overall proportion of students who support the People's Park Project without having been exposed to the information on page 14. Interpret the interval in the context of the problem in a clear sentence. Does your point estimate approximately match that reported in the Chancellor's email?
+
+#### Question 10
+
+Select a confidence level *lower* than 95% and recalculate the interval from the previous question to express this confidence level. In general, are intervals with higher confidence wider or narrower than those with lower confidence?
+
```

Comment:
Here's the other new question. Very straight-forward but gives them experience thinking about difference confidence levels.